lldb/test/Shell/SwiftREPL/RedirectInputUnreadable.test

```diff
@@ -9,3 +9,4 @@
 
 < A.swift
 // LLDB: could not read file at path 'A.swift'
+// RUN: rm -f A.swift
```

Comment:
That looks sketchy. If we create any filed during a test they should be given a unique name containing `%t` or be in a directory that contains `%t`.
Can you fix that to say `< %t.swift`, etc?

Reply:
Otherwise we might overwrite some other test's A.swift when running the tests in parallel...

Reply:
thanks, I  changed it to `%t/A.swift`